libcaf_net/caf/net/http/async_client.test.cpp

```diff
@@ -115,7 +113,7 @@ struct fixture {
     auto client = net::http::client::make(std::move(app));
     auto transport = net::octet_stream::transport::make(fd2, std::move(client));
     auto mgr = net::socket_manager::make(mpx.get(), std::move(transport));
-    mpx->start(mgr);
+    std::ignore = mpx->start(mgr);
```

Comment:
Shouldn't the test abort here instead? This isn't in a test scope, but it could still do something like `if (!...) { throw std::logic_error(...); }`